README.md

```diff
@@ -172,6 +172,12 @@ yesno.matching({ request: { path: '/post' } }) .respond((request) => ({ statusCo
 
 Responses defined in this way take precedence over normally loaded mocks.
 
+### Ignoring matched mocks to proxy requests
```

Comment:
If I have these calls 
A, 
B,
C,
B
And I want to allow B to be a live call both times, will I net to have two `.ignores` or one?